web-playground/src/App.tsx

```diff
@@ -28,9 +35,20 @@ function App() {
         }
     }
 
+    function changeLanguage(l: string) {
+        if (languages[l]) {
+            setInput(languages[l].input);
+            setQuery(languages[l].query);
+        }
+    }
+
     return (
         <div className="App">
             <div className="header">
+                <select onChange={e => changeLanguage(e.target.value)}>
+                    <option value="">Choose a language</option>
```

Comment:
```suggestion
                    <option value="">Choose an example language</option>
```

I've added "example" here to make it feel more playground-like. Otherwise, it comes across -- despite it not being true -- as a bit "immutable".

(I'm not convinced that "example" is the best word...)

Reply:
I've used "reference language" in the new version, to suggest that modifications can be added